media/css/firefox/family-nav.less

```diff
@@ -4,6 +4,10 @@
 
 @import "../sandstone/lib.less";
 
+#tabzilla:before {
+    background-color: @textColorSecondary;
+}
```

Comment:
Yeah I think this was my first edits when I missed some dependencies. I removed it.